tests/correct/incrementAndDecrement.rock

```diff
@@ -1,3 +1,5 @@
+(This part tests that increment and decrement works on integers,
+including multi increment and decrements with optional commas)
```

Comment:
Should multiline comments even work? Anyway with the change below adding one on line 5 the second line can be cut anyway.